### DISCUSSION TITLE
Module @trpc/server has not exported member initTRPC

### DISCUSSION THREAD

**Discussion by chandantiwari1** [trpc/trpc#3001] [General]
This error is occuring with the latest version of trpc which 9.27.4 .

**Comment by nelsonsvo** [comment]
try with the new version 10 beta

**Reply by maietta** [reply]
Running in the same issue, yet again.

@tanstack/svelte-query": "^4.24.6
@trpc/client": "^10.11.1
@trpc/server": "^10.11.1
trpc-sveltekit": "^3.2.9
